src/types/markers.js

```diff
@@ -48,6 +48,11 @@ export type PaintProfilerMarkerTracing = ProfilerMarkerTracing & {
     | 'Composite',
 };
 
+/*
+ * Note that the phase times for the nursery are in Microseconds, while the
+ * tenured heap uses Milliseconds.
+ */
+export type MinorPhaseTimes = { [phase: string]: Microseconds };
```

Comment:
should https://github.com/devtools-html/perf.html/blob/master/src/profile-logic/convert-markers.js#L53 return `MinorPhaseTimes` ?

Reply:
Note: Flow now supports "opaque" types, see https://flow.org/en/docs/types/opaque-types/, the version of flow we use is recent enough for this.
So likely we should use these opaque types for Microseconds and Milliseconds so that such errors are actually found by Flow. Not for you to do in this patch, but I thought you may be interested :)

Reply:
I wondered about opaque types.  That would be good to use.